### PR TITLE
UIIN-1098 Render trigger button conditionally

### PR DESCRIPTION
**Pull request by uladislausamets**
## Purpose
https://issues.folio.org/browse/UIIN-1098

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [ ] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [ ] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: PluginFindRecord/PluginFindRecord.js [hunk 1]
```diff
@@ -99,6 +99,7 @@ PluginFindRecord.propTypes = {
   marginBottom0: PropTypes.bool,
   marginTop0: PropTypes.bool,
   renderTrigger: PropTypes.func,
+  withTrigger: PropTypes.bool,
```
Comment (NikitaSedyx) [inline PluginFindRecord/PluginFindRecord.js:102]: please make it default true

File: PluginFindRecord/PluginFindRecord.js [hunk 2]
```diff
@@ -34,9 +34,10 @@ class PluginFindRecord extends React.Component {
     openModal: true,
   });
 
-  closeModal = () => this.setState({
-    openModal: false,
-  });
+  closeModal = () => {
+    this.props.onClosePluggin();
```
Comment (NikitaSedyx) [inline PluginFindRecord/PluginFindRecord.js:38]: as I see it's optional, what if it's not passed?

Comment (NikitaSedyx) [reply PluginFindRecord/PluginFindRecord.js:38]: just `onClose`

File: test/bigtest/tests/find-instance-multiselect-test.js [hunk 3]
```diff
@@ -66,4 +57,14 @@ describe('Find instance plugin with multiselect', function () {
       });
     });
   });
+
+  describe('close plugin', function () {
```
Comment (NikitaSedyx) [inline test/bigtest/tests/find-instance-multiselect-test.js:61]: it doesn't test passed prop is called

File: test/bigtest/tests/find-instance-multiselect-test.js [hunk 4]
```diff
@@ -9,31 +9,22 @@ const INSTANCES_COUNT = 15;
...
+  setupApplication({
+    isMultiSelect: true,
+  });
 
   beforeEach(async function () {
     this.server.createList('instance', INSTANCES_COUNT);
   });
 
   describe('Find instance button', () => {
-    it('should be rendered', function () {
```
Comment (aliaksei-chumakou) [inline test/bigtest/tests/find-instance-multiselect-test.js:19]: it looks like nothing should be changed with the same props, and here is the question - why tests should be deleted?

**Pull request merged**